.github/workflows/create-ephemeral-resources.yml

```
@@ -83,14 +85,24 @@ jobs:
   
       - 
         # TODO: Move this step to /k8s-manifests/k8s-rbac/
+        # Source: https://claudm.github.io/codepipeline/configmap/
         name: allow admins to access eks control plane
         run: |
-          kubectl patch configmap/aws-auth -n kube-system --type merge -p "$(cat <<EOF
-          data:
-            mapRoles: |
-              - rolearn: arn:aws:iam::${{ env.AWS_ACCOUNT }}:role/${{ env.ADMINS_ROLE }}
-                username: admin
-                groups:
-                  - system:masters
-          EOF
-          )"
+          ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
```

Can we put that in a separated script for better visualization?
```suggestion
          ./scripts/patch-configmap.sh
```

---

It would be good if we needed this script later, but we'll not use it anywhere else and it would create complexity in understanding the job's step. Besides that, we would increase the complexity of our current folder and file structure as well

---

I think it is better to maintainability. But that's fine